website/docs/r/vapp_vm.html.markdown

```diff
@@ -97,13 +116,29 @@ The following arguments are supported:
 * `cpu_cores` - (Optional; *v2.1+*) The number of cores per socket
 * `metadata` - (Optional; *v2.2+*) Key value map of metadata to assign to this VM
 * `initscript` (Optional) A script to be run only on initial boot
-* `network_name` - (Optional) Name of the network this VM should connect to
-* `vapp_network_name` - (Optional; *v2.1+*) Name of the vApp network this VM should connect to
-* `ip` - (Optional) The IP to assign to this vApp. Must be an IP address or
-  one of "dhcp", "allocated" or "none". If given the address must be within the
+* `network_name` - (Optional; **Deprecated** by `networks`) Name of the network this VM should connect to.
+* `vapp_network_name` - (Optional; v2.1+; **Deprecated** by `networks`) Name of the vApp network this VM should connect to.
+* `ip` - (Optional; **Deprecated** by `networks`) The IP to assign to this vApp. Must be an IP address or
+one of dhcp, allocated or none. If given the address must be within the
   `static_ip_pool` set for the network. If left blank, and the network has
   `dhcp_pool` set with at least one available IP then this will be set with
-  DHCP.
+DHCP.
+* `networks` - (Optional; *v2.2+*) List of network interfaces to attach to VM. **Deprecates**: `network_name`, `ip`, `vapp_network_name`. **Note**: all params of this parameter and itself do force recreation of VMs!
```

Comment:
"all params of this parameter and itself" should become "this property and all its parameters"